src/containers/explore-product/index.tsx

```diff
@@ -20,12 +21,13 @@ const LoadingSpinner = () => (
 const ProductArea = ({ className, space, nfts }: Props) => {
   const [hasMore, setHasMore] = useState(true);
 
-  const loadMoreItems = () => {
-    if (nfts.length >= 10) {
+  const loadMoreItems = async () => {
+    const { data } = await axios.get('http://localhost:3334/v1/items?limit=10');
+    if (nfts.length >= data.itemsLength) {
       setHasMore(false);
     }
-
-    console.log('finish');
+    console.log(data.items);
+    console.log(data.itemsLength);
```

Comment:
Remove console.log

Reply:
Where did you take this code? This component was removed